jaeger-core/src/test/java/io/jaegertracing/internal/JaegerSpanContextTest.java

```diff
@@ -57,10 +35,5 @@ public void testToStringFormatsPositiveFields() {
 
     assertEquals(
         "fffffffffffffff6:fffffffffffffff6:fffffffffffffff6:81", context.contextAsString());
-    JaegerSpanContext contextFromStr = TextMapCodec.contextFromString(context.contextAsString());
-    assertEquals(traceId, contextFromStr.getTraceId());
-    assertEquals(spanId, contextFromStr.getSpanId());
-    assertEquals(parentId, contextFromStr.getParentId());
-    assertEquals(flags, contextFromStr.getFlags());
```

Comment:
don't see how removing this makes the test equivalent. But this whole test can move to text codec test, since it's not really testing the context toString

Reply:
Right let me fix that.